src/features/audit-log/AuditLogTable.stories.tsx

```
@@ -0,0 +1,211 @@
+    totalCount: 0,
+    isLoading: false,
+    error: null,
+  },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    await expect(canvas.findByText(/no audit log entries found/i)).resolves.toBeInTheDocument();
+  },
+};
+
+/** Error state when the client provides an error message. */
+export const Error: Story = {
```

Biome (2.4.7)

[error] Do not shadow the global "Error" property.

(lint/suspicious/noShadowRestrictedNames)